packages/wrangler/src/config/environment.ts

```diff
@@ -877,4 +877,11 @@ export type ExperimentalAssets = {
 	/** Absolute path to assets directory */
 	directory: string;
 	binding?: string;
+	serve_exact_matches_only?: boolean;
+	trailing_slashes?: "auto" | "add" | "remove";
+	not_found_behavior?:
+		| "default"
+		| "single-page-application"
+		| "404-page"
```

Comment:
Feels like serve_exact_matches is duplicating config from the other two. How about:

```
asset_path_rewriting?: AssetPathRewritingConfig
```

where

```
type AssetPathRewritingConfig = {
	trailing_slashes?: "auto" | "add" | "remove";
	not_found_behavior?:
		| "default"
		| "single-page-application"
		| "404-page"
}
```

Reply:
`not_found_behavior` is currently independent of `serve_exact_matches_only`. If you want to bring `not_found_behavior` in as a child of `serve_exact_matches_only` and can validate that it still meets Angular's needs, then I'd again be happy to make this change.

Reply:
Hmm. Good question. I assumed that returning a 404 is a kind of rewrite of the request.